app/models/subject_specialism.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class SubjectSpecialism < ApplicationRecord
+  belongs_to :allocation_subject
```

Comment:
```suggestion
  belongs_to :allocation_subject, inverse_of: :subject_specialisms
```

Reply:
Commited in the 1994-add-allocation-subjects-and-specialism-subjects branch 👍